lib/gen_stage.ex

```diff
@@ -718,8 +718,12 @@ defmodule GenStage do
   @typedoc "The supported stage types."
   @type type :: :producer | :consumer | :producer_consumer
 
-  @typedoc "The supported init options."
-  @type options :: keyword()
+ @typedoc "Option values used by the `init*` functions"
+  @type option :: {:max_demand, non_neg_integer} |
+                  {:min_demand, non_neg_integer}
```

Comment:
We have many more options than that. Please check the docs for the init callback? min_demand and max_demand are subscription options, not init ones. :)

Reply:
@josevalim Yup, you were right. Fixed now. I've noticed sometimes `term` is used and sometimes `any` is used. 
Same issues whenever a module is expected where I see `atom` and sometimes `module`. 

Could you explain the difference? 
Also, how do you feel about paranthesis, do you expect them for types that could be "generic", e.g `keyword()` without the contained type?